.github/workflows/ci.yml

```
@@ -13,6 +13,15 @@ jobs:
             - uses: actions/setup-node@v2
               with:
                   node-version: 14
+            - name: Get yarn cache directory path
+              id: yarn-cache-dir-path
+              run: echo "::set-output name=dir::$(yarn cache dir)"
+            - uses: actions/cache@v2
+              with:
+                path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+                key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
```

I wonder which `yarn.lock` file this will eventually point to. There are three in this repository:
- `<root>/yarn.lock`
- `<root>/repositories/yarn.lock`
- `<root>/ci/yarn.lock` (Not committed to remote as using latest packages is intended)

It would be ideal if it picked both the root one and `repositories` one.

---

It should be picking all of them.

To make it more explicit, one could probably replace iy with: 

```suggestion
                key: ${{ runner.os }}-yarn-${{ hashFiles('./yarn.lock', './repositories/yarn.lock') }}
```

---

Should be fine as it is:

> Returns a single hash for the **set of files** that matches the path pattern.

https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#hashfiles